<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1072</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1072</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Trader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>1061</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>계좌 현황</string>
     </attribute>
     <widget class="QTableView" name="accountInfoTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>자동매매 현황</string>
     </attribute>
     <widget class="QTableView" name="autoTradeInfoTableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1051</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>매수 설정</string>
    </property>
    <widget class="QComboBox" name="buyConditionComboBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>221</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>매수 조건식:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="buyAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>200,000</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>매수 금액:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="marketBuyRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>시장가 매수</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="limitBuyRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>현재가 대비</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="limitBuySpinBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>69</width>
       <height>23</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>호가로 지정가 주문</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>601</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>매도 설정</string>
    </property>
    <widget class="QDoubleSpinBox" name="stopLossDoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>60</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>-2.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="trailingStopDoubleSpinBox1">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>90</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ㄴ 트레일링 발동 이후 고가 대비 등락률(%)이</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>% 이하로 내려가면 전량 매도 주문</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="trailingStopCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>평균단가 대비 현재 수익률이</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="trailingStopDoubleSpinBox2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-99.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>-1.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="stopLossCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>191</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>평균단가 대비 현재 수익률이</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>% 이상으로 올라가면 트레이링 발동</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>매도 조건식:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sellConditionComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>% 이하로 내려가면 전량 매도 주문</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="limitSellRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>현재가 대비</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="limitSellSpinBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>69</width>
       <height>23</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QRadioButton" name="marketSellRadioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>시장가 매도</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>호가로 지정가 주문</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>초 이상 미체결 발생 시 시장자 정정 주문</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="amendOrderSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>미체결 관리</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>최대 자동매매 종목 수:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="QPushButton" name="autoOnPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>자동매매 ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="autoOffPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>자동매매 OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savePushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>설정 저장</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
